packages/mcp-server/src/tool-handlers/messages/search-messages-handler.ts

```
@@ -0,0 +1,158 @@
+import { CustomHandlerFunction, asFormattedMCPContentResult } from '../types';
```

ESLint

[error] 'asFormattedMCPContentResult' is defined but never used.

(unused-imports/no-unused-imports)